static/app/views/detectors/components/forms/metricFormData.tsx

```diff
@@ -0,0 +1,311 @@
+import {useFormField} from 'sentry/components/workflowEngine/form/useFormField';
+import {PriorityLevel} from 'sentry/types/group';
+import type {
+  DataCondition,
+  DataConditionGroup,
+} from 'sentry/types/workflowEngine/dataConditions';
+import {
+  DataConditionGroupLogicType,
+  DataConditionType,
+} from 'sentry/types/workflowEngine/dataConditions';
+import type {Detector} from 'sentry/types/workflowEngine/detectors';
+import {
+  AlertRuleSensitivity,
+  AlertRuleThresholdType,
+} from 'sentry/views/alerts/rules/metric/types';
+
+interface PrioritizeLevelFormData {
```

Comment:
could we build this interface dynamically based on the `PriorityLevel` enum? that way we can ensure that the key is always something from priority level, then the value is the threshold set by the user, rather than trying to map `highThreshold` to `75` later.

Reply:
i get what you're saying but with forms we're working with a flat k/v object